Sources/TuistAutomation/XcodeBuild/XcodeBuildController.swift

```diff
@@ -213,6 +213,13 @@ public final class XcodeBuildController: XcodeBuildControlling {
             .asSingle()
     }
 
+    public func version() -> Observable<String> {
```

Comment:
Is there any reason why we cannot use `System.swiftVersion()`? It has the caching of command (to avoid launch of multiple processes) and it seems we use to hash the cache of manifest as well.

Reply:
Didn't know of that, refactored to use that instead 🚀